src/Sylius/Bundle/ShopBundle/Controller/ContactController.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\ShopBundle\Controller;
+
+use Sylius\Bundle\CoreBundle\EmailManager\ContactEmailManager;
+use Sylius\Bundle\CoreBundle\Form\Type\ContactType;
+use Sylius\Component\Channel\Context\ChannelContextInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @author Grzegorz Sadowski <grzegorz.sadowski@lakion.com>
+ */
+class ContactController extends Controller
```

Comment:
When you extend default controller your controller is container aware so you don't need to inject your dependecies. Imo you shoud not extend it :dancer: .